rust/src/nlp/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "nlp"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+rust-bert = "=0.5.3"
```

Comment:
This seems wrong.

Reply:
ooh okay thanks @liangwen12year  does it mean I have to place the dependency else where

Reply:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html, `rust-bert = "=0.5.3"`, this seems to be  invalid ?

Reply:
commiting correct definition upstream now

Reply:
`rust-bert = "0.5.3"` ?

Reply:
yah right